library/ecjpake.c

```diff
@@ -820,6 +820,8 @@ static const unsigned char ecjpake_test_password[] = {
     0x65, 0x73, 0x74
```

Comment:
Commit message: the commit message is too long and not rendered properly by GitHub.

Reply:
fixed